src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -93,101 +100,136 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
     projectNumber: changeRequest.wbsNum.projectNumber,
     workPackageNumber: 0
   });
+  const [reviewerIds, setReviewerIds] = useState<number[]>([]);
+  const { data: users, isLoading: isLoadingAllUsers, isError: isErrorAllUsers, error: errorAllUsers } = useAllUsers();
+  const { mutateAsync: requestCRReview } = useRequestCRReview(changeRequest.crId.toString());
+  const toast = useToast();
+  const currentUser = useCurrentUser();
+  const history = useHistory();
   if (isError) return <ErrorPage message={error?.message} />;
-  if (!project || isLoading) return <LoadingIndicator />;
+  if (isErrorAllUsers) return <ErrorPage message={errorAllUsers?.message} />;
+  if (!project || isLoading || isLoadingAllUsers || !users) return <LoadingIndicator />;
+
   const { name: projectName } = project;
 
-  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
-    setAnchorEl(event.currentTarget);
+  const handleRequestReviewerClick = async () => {
+    if (reviewerIds.length === 0) return;
```

Comment:
you should toast an error before returning here saying: "Must select at least one reviewer to request review from"